tests/test_plot.py

```diff
@@ -400,3 +409,28 @@ def test_scatter_plot_2d():
     scatter_plot_2d(ax, data_x, data_y, ymax=ymax)
     assert(ax.get_ylim()[1] <= ymax)
     plt.close()
+
+
+@pytest.mark.parametrize('sigmas', [('1sigma', 0.682689492137086),
+                                    ('2sigma', 0.954499736103642),
+                                    ('3sigma', 0.997300203936740),
+                                    ('4sigma', 0.999936657516334),
+                                    ('5sigma', 0.999999426696856)])
+def test_quantile_plot_interval_str(sigmas):
+    q1, q2 = quantile_plot_interval(q=sigmas[0])
+    assert q1 == 0.5 - sigmas[1] / 2
+    assert q2 == 0.5 + sigmas[1] / 2
+
+
+@pytest.mark.parametrize('floats', [0, 1, 0.1, 0.9])
+def test_quantile_plot_interval_float(floats):
+    q1, q2 = quantile_plot_interval(q=floats)
+    assert q1 == min(floats, 1 - floats)
+    assert q2 == max(floats, 1 - floats)
+
+
+@pytest.mark.parametrize('q1, q2', [(0, 1), (0.1, 0.9), (0, 0.9), (0.1, 1)])
+def test_quantile_plot_interval_tuple(q1, q2):
+    _q1, _q2 = quantile_plot_interval(q=(q1, q2))
+    assert _q1 == q1
+    assert _q2 == q2
```

Comment:
These parameterised tests are very neat! It would be great to update the rest of the suite.

Reply:
Agreed, but that will have to be for a future PR.